docs/usage/from-a-terminal.rst

```diff
@@ -1179,7 +1184,7 @@ Global overview
                                 Sets the command to execute before the final commit.
                                 Configured value: None
         --ci-commit-message CLI_TESTING__CI__COMMIT_MESSAGE, --commit-autosave-message CLI_TESTING__CI__COMMIT_MESSAGE
-                                Sets the commit message to apply everytime we have
+                                Sets the commit message to apply every time we have
```

Comment:
Please also apply the change at:
https://github.com/funilrys/PyFunceble/blob/0d5e5450473aff6712277d8dd0bbc240e2f953f1/PyFunceble/cli/entry_points/pyfunceble/cli.py#L868-L869